R/import_data.R

```diff
@@ -26,16 +26,15 @@ isColumnTypesCorrect <- function(col.types){
 
 #' Checks if the complete file was read or not.
 #' 
-#' \code{isFullFile} returns a logical scalar after checking 
+#' \code{isPreview} returns a logical scalar after checking 
 #' the attributes of the dataframe sent as an argument. 
-#' Returns TRUE if it is the complete file. Returns FALSE if 
-#' it is a partial file read. 
+#' Returns TRUE if it was read as a preview. 
 #' 
 #' @param user.data.frame the dataframe returned after reading the file
 #' @return A logical scalar
-isFullFile <- function(user.data.frame){
+isPreview <- function(user.data.frame){
   
-  if (attr(user.data.frame, "full.file") == TRUE){
+  if (attr(user.data.frame, "preview") == TRUE){
```

Comment:
Needs to also check existence of the attribute (you can use `&&`):
```{R}
if (!is.null(attr(user.data.frame, "preview")) && attr(user.data.frame, "preview")) return TRUE
```

Note: `if (x == TRUE)` is exactly the same as `if (x)` by definition of `if`.